404.md

```diff
@@ -1,14 +1,18 @@
 ---
 layout: page
+title: Error 404
 description: "Page not found. Your pixels are in another canvas."
 permalink: /404.html
 ---
-<div class="text-center">
-<h1>404: Not Found</h1>
+
 The tumbeasts are to be blamed, as your destination couldn't be found.
-</div>
+
 <figure>
-<img src="{{ site.url }}/assets/images/bg-arrow.png" alt="down-arrow">
+  <amp-img src="{{ site.url }}/assets/images/bg-arrow.png"
```

Comment:
This was purely to fix the fact that I had a non-AMP compliant 404 page; not terribly high priority, but clutters up this commit.